src/Client/QueryFuzzer.cpp

```diff
@@ -991,6 +1084,10 @@ void QueryFuzzer::fuzz(ASTPtr & ast)
     {
         fuzzCreateQuery(*create_query);
     }
+    else if (auto * explain_query = typeid_cast<ASTExplainQuery *>(ast.get()))
+    {
+        fuzzExplainQuery(*explain_query);
```

Comment:
Ok. Btw, we can also add various types of EXPLAIN to a SELECT query under certain probability. Let's make it in a subsequent PR.